pyproject.toml

```
@@ -25,7 +25,7 @@ packages = [{ include = "pineko", from = "src" }]
 [tool.poetry.dependencies]
 python = ">=3.8,<3.12"
 eko = "^0.13.2"
-pineappl = { version = "^0.6.0a17", allow-prereleases = true }
+pineappl = { version = "^0.6.2", allow-prereleases = true }
```

```suggestion
pineappl = "^0.6.2"
```
You do not need a pre-release any longer.

---

0d28bbc